src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/DotNetHelper.cs

```diff
@@ -234,10 +235,8 @@ private static string GetBinLogOption(string projectName, string command, string
         return $"/bl:{Path.Combine(LogsDirectory, $"{fileName}.binlog")}";
     }
 
-    private static bool DetermineIsMonoRuntime()
+    private static bool DetermineIsMonoRuntime(string dotnetRoot)
```

Comment:
Why does `dotnetRoot` have to be passed in?

Reply:
It makes more visible at the caller site we're passing in the directory that got just extracted too.

Reply:
Should I make a change here?

Reply:
I am fine with your original proposal.